src/main/java/com/jakewharton/rxrelay/Relay.java

```diff
@@ -26,6 +26,22 @@ protected Relay(OnSubscribe<R> onSubscribe) {
   }
 
   /**
+   * Portrays a object of an Action1 implementation as a simple Relay object. This is useful, for instance,
```

Comment:
Is this copied from `asObservable()`? The first sentence reads really weird, almost backwards.